src/apps/service/apps/jobs/sharing.clj

```diff
@@ -148,7 +155,8 @@
 
 (defn- process-job-inputs
   [f apps-client job]
-  (remove nil? (map f (list-job-inputs apps-client job))))
+  (otel/with-span [s ["process-job-inputs"]]
+    (doall (remove nil? (map f (list-job-inputs apps-client job))))))
```

Comment:
When I was testing, my input file wasn't getting shared. Adding this `doall` fixed it -- I think that the function wasn't actually getting called for inputs properly. Also added it to `process-child-jobs` above.